### ISSUE TITLE
no id prefix for abilities/traits

### ISSUE THREAD

**Issue by sevu** [ProditorMagnus/Ageless-for-1-14#15]
in difference to race and unit type, they do not need a prefix.

Abilites and traits are dumped by macros into the units, not called by id.
A few one use events for their implementation, which check for the id, however, these events are inserted either by the era or the unit type, and thus cause no problems either.

But the prefix has ir may have an disadvantage: The help may list the AE and EoMa version. (or may not?)

TL;DR: getting that changed in the porting script at some point

related, but different: https://github.com/inferno8/wesnoth-Era_of_Magic/issues/27

**Comment by ProditorMagnus** [comment]
Distinct id might help SXC for removing them.

**Comment by sevu** [comment]
In case they do, SXC wouldn't want to differentiate between the AE and EoMa version though.

**Comment by ProditorMagnus** [comment]
Specific issues can be reported, not planning focused effort on all cases. Abilities with events are more likely to be adjusted.
